tools/runme/build.sh

```diff
@@ -111,6 +111,21 @@ _sbt_build() {
   cd "$owd"
 }
 
+_sbt_adb_notebooks() {
+  show section "Running ADB Notebooks"
+  if [[ "$BUILDMODE" = "server" ]]; then
+    local token="$(__ az keyvault secret show --vault-name mmlspark-keys --name adb-token)"
+    token="${token##*\"value\": \"}"; token="${token%%\"*}"
+    export MML_ADB_TOKEN="$token"
+  fi
+
+  local owd="$PWD"
+  cd "$SRCDIR"
+  local TESTS="$TESTS"
```

Comment:
This line should be redundant: all it does is ensure that `TESTS` are not modified by `_sbt_run`, and that shouldn't happen.

Reply:
thanks!